homeassistant/components/unifi/sensor.py

```diff
@@ -172,8 +172,8 @@ def name(self) -> str:
     def native_value(self) -> datetime:
         """Return the uptime of the client."""
         if self.client.uptime < 1000000000:
-            return (dt_util.now() - timedelta(seconds=self.client.uptime)).isoformat()
-        return dt_util.utc_from_timestamp(float(self.client.uptime)).isoformat()
+            return dt_util.now() - timedelta(seconds=self.client.uptime)
```

Comment:
Side note: The timestamp should always be in UTC time. Looks like we're returning local time here.